src/shared/modules/discovery/discoveryDuck.js

```diff
@@ -88,7 +92,13 @@ export const discoveryOnStartupEpic = (some$, store) => {
     })
     .mergeMap(action => {
       if (action.forceURL) {
-        store.dispatch(updateDiscoveryConnection({ host: action.forceURL }))
+        store.dispatch(
+          updateDiscoveryConnection({
+            username: action.username,
+            password: action.password,
```

Comment:
I think we only want to update`username` and `password` if they are present in the action object. 
If not, they should be left as they were.